src/map/lua/lua_baseentity.cpp

```diff
@@ -10383,6 +10383,35 @@ bool CLuaBaseEntity::delLatent(uint16 condID, uint16 conditionValue, uint16 mID,
     return static_cast<CCharEntity*>(m_PBaseEntity)->PLatentEffectContainer->DelLatentEffect(conditionID, conditionValue, modID, modValue);
 }
 
+/************************************************************************
+ *  Function: getEquippedItemsMaxModValue()
+ *  Purpose : Returns the highest integer value of a specified Mod on all equiped items
+ *  Example : local maxValue = player:getEquippedItemsMaxModValue(xi.mod.GEOMANCY_BONUS)
+ *  Notes   :
+ ************************************************************************/
+
+int16 CLuaBaseEntity::getEquippedItemsMaxModValue(Mod modId)
```

Comment:
kinda long name. `getMaxGearMod()` ?

Reply:
related: do we mistakenly stack movement speed from gear? we must if we didn't already have a function to do this.

Reply:
Yep, gets clamped is all

Reply:
That’s a better name, I like getMaxGearMod, I’ll get that changed if others agree

Reply:
Sounds good to me

Reply:
ok changed to getMaxGearMod()